src/Pages/ResultsFixtures/View.elm

```diff
@@ -77,6 +77,21 @@ dayResultsFixtures responsive leagueGamesForDay =
         ]
         (List.map (gameRow responsive) leagueGamesForDay.games)
 
+formatPlayersCount : List (String, Int) -> List String
+formatPlayersCount =
+            List.map
+                (\( playerName, timesScored ) ->
+                    if timesScored <= 1 then
+                        playerName
+
+                    else
+                        playerName ++ " (" ++ String.fromInt timesScored ++ ")"
+                )
+
+gameRowScorers : List String -> String
```

Comment:
I think I would probably store the `playerOccurrences` in the model, which keeps the calculations and view separate and reduces the complication in this function a little. It's not a big deal though, especially as the calculation function is already in a calculation module.

Something for another time ...